yaptide/admin/simulators.py

```diff
@@ -115,5 +227,34 @@ def install(**kwargs):
         click.echo(f'Simulator {sim_type.name} installed')
 
 
+@run.command
+@click.option('--bucket', help='S3 bucket name')
+@click.option('--file', help='Path to file to upload')
+@click.option('--endpoint', envvar='S3_ENDPOINT', default=endpoint, help='S3 endpoint')
+@click.option('--access_key', envvar='S3_ACCESS_KEY', default=access_key, help='S3 access key')
+@click.option('--secret_key', envvar='S3_SECRET_KEY', default=secret_key, help='S3 secret key')
+@click.option('--encryption_key', envvar='S3_ENCRYPTION_KEY', default=encryption_key, help='S3 encryption key')
```

Comment:
```suggestion
@click.option('--encryption_key', envvar='S3_ENCRYPTION_KEY', default=encryption_key, help='binary encryption key')
```

Fernet has strict rules on how the key should look like (i.e. string that is converted to extactly 32 bytes). Either provide a clear instruction how to generate such key (random selection of letters doesn't work) or swithc to a different approach, i.e. using passwords in Fernet